src/app/modules/core/advertisement/components/card/card-top/check-in-check-out/check-in-check-out.component.spec.ts

```diff
@@ -1,4 +1,5 @@
 import { ComponentFixture, TestBed, waitForAsync } from '@angular/core/testing'
+import { FormBuilder } from '@angular/forms'
 
 import { CheckInCheckOutComponent } from './check-in-check-out.component'
```

Comment:
@modules/core/advertisement/components/card/card-top/check-in-check-out/check-in-check-out.component

Reply:
Se i files sono nella stessa cartella, direi che il percorso relativo va ancora bene in quanto, nel caso di refactoring.....quasi sicuramente si spostera' l'intera cartella e quindi il percorso sarebbe ancora valido. Grazie.